tests/auth_tests/test_basic.py

```diff
@@ -44,11 +45,18 @@ def test_user(self):
         self.assertEqual(u.get_username(), 'testuser')
 
         # Check authentication/permissions
-        self.assertTrue(u.is_authenticated())
+        self.assertFalse(u.is_anonymous)
+        self.assertTrue(u.is_authenticated)
         self.assertFalse(u.is_staff)
         self.assertTrue(u.is_active)
         self.assertFalse(u.is_superuser)
 
+        # Backwards-compatibility callables
+        with self.assertRaises(RemovedInDjango20Warning):
+            self.assertFalse(u.is_anonymous())
```

Comment:
The inner assertions won't run because the exception will be raised before they do. You'll need to use `django.test.ignore_warnings()` to test the functionality. Also, it's a good idea to use `assertRaisesMessage` to check the message as well.